src/main/java/com/Saurabh/GitHubAiAPI/StringOperations.java

```diff
@@ -43,7 +43,7 @@ public static void main(String[] args) {
 
         // Validate input
```

Comment:
There is a spelling mistake in the output message. It should be 'Invalid input. Please enter a non-empty string.'